cmd/check.go

```diff
@@ -90,5 +100,4 @@ func init() {
 	flags.StringSliceVar(&checkflags.AuditPlan, "auditplan", defaultAuditPlan, "audit plan is the ordered list of operator test functions to be called during a capability audit.")
 	flags.BoolVar(&checkflags.ListPackages, "list-packages", false, "list packages in the catalog")
 	flags.StringSliceVar(&checkflags.FilterPackages, "filter-packages", []string{}, "a list of package(s) which limits audits and/or other flag(s) output")
-	flags.BoolVar(&checkflags.AllInstallModes, "all-installmodes", false, "when set, all install modes supported by an operator will be tested")
```

Comment:
Take note of any functionality removed. Here I can see the functionality to test all install modes has been removed.

Create a new issue if you want this reworked at a future time.

Reply:
Totally agree. Here is the new issue: Enable all supported install modes testing #189

Reply:
@acmenezes , regarding the name convention for parameters, should we create them with the `-` (i.e `--list-packages`) or not like `--auditplan`?

Reply:
There we go 9929e85 - dash included.